src/main/scala/gpp/ui/forms/EnumViewSelect.scala

```diff
@@ -106,112 +101,23 @@ final case class EnumViewSelect[F[_], A](
   val enum:             Enumerated[A],
   val display:          Display[A],
   val effect:           Effect[F]
-) extends ReactProps[EnumViewSelect[Any, Any]](EnumViewSelect.component) {
-  def withMods(mods: TagMod*): EnumViewSelect[F, A] = copy(modifiers = modifiers ++ mods)
-}
+) extends ReactProps[EnumViewSelectBase](EnumViewSelectBase.component)
+    with EnumViewSelectBase {
+
+  type AA    = A
+  type BB    = A
+  type FF[X] = F[X]
```

Comment:
I think you can do `type FF[_] = F[_]` here.

Reply:
No, that resulted in:

> F <: EnumViewMultipleSelect.this.FF, but class ViewF is invariant in type F.

Reply:
I think the compiler was interpreting `FF[_] = F[_]` as saying `FF` and `F` were the same type even if their type parameters were different. Which seem strange. `FF[X] = F[X]` says they are the same type as long as they have the same type parameter.